app/src/main/java/com/dessalines/thumbkey/keyboards/DEMessageEaseSymbols.kt

```diff
@@ -15,6 +29,132 @@ import com.dessalines.thumbkey.utils.KeyboardC
 import com.dessalines.thumbkey.utils.KeyboardDefinition
 import com.dessalines.thumbkey.utils.KeyboardDefinitionModes
 import com.dessalines.thumbkey.utils.SwipeDirection
+import com.dessalines.thumbkey.utils.SwipeNWay
+
+val KB_DE_MESSAGEEASE_SYMBOLS_NUMERIC_KEY_ITEM =
```

Comment:
Please don't use a custom numeric key, or you will lose any functionality we add later on.

Reply:
I've tried to make it as close to MessagEase as possible.